LuaMenu/widgets/chobby/components/downloader.lua

```diff
@@ -205,42 +205,12 @@ function Downloader:DownloadProgress(listener, downloadID, downloaded, total)
 
 	self._lastUpdate = currentTime
 
-	local elapsedTime = os.clock() - self.downloads[downloadID].startTime
```

Comment:
What is this all about?

Reply:
It's mainly deletion of code that is never used. Why calculate various properties if we're not going to display it.